src/interfaces/projectInterfaces.ts

```diff
@@ -148,26 +148,26 @@ export interface IProjectRequestObject {
 }
 
 export enum ProjectType {
-  ProjectComplex = 'projectComplex',
-  Street = 'street',
-  Traffic = 'traffic',
-  Sports = 'sports',
-  Omastadi = 'omaStadi',
-  ProjectArea = 'projectArea',
-  Park = 'park',
+  ProjectComplex = 'a86ecdd0-d53e-45fc-95bf-d9ddcf82c8ba',
+  Street = '2d5a2f44-2dc1-4f3a-8616-40d0d6d7e6fc',
+  Traffic = 'dafdbe4f-2d89-4c5a-9be4-cef00b3c1ce4',
+  Sports = '262aef64-7704-448b-b434-6973570ac22e',
+  Omastadi = '9defdb0d-eea4-40fc-b283-30e1be14e163',
+  ProjectArea = '05552ebb-aec8-4ca7-b2e1-dea66b0b07ce',
+  Park = 'bcfe8874-5de1-49e8-a5b1-45b363a28222',
 }
 
 export enum ProjectPhase {
-  Proposal = 'proposal',
-  Design = 'design',
-  Programming = 'programming',
-  DraftInitiation = 'draftInitiation',
-  DraftApproval = 'draftApproval',
-  ConstructionPlan = 'constructionPlan',
-  ConstructionWait = 'constructionWait',
-  Construction = 'construction',
-  WarrantyPeriod = 'warrantyPeriod',
-  Completed = 'completed',
```

Comment:
Don't use uuid:s because they are environment based which means uuid:s are different in different environments

Reply:
Resolved